### DISCUSSION TITLE
New Epsilon TV

### DISCUSSION THREAD

**Discussion by Playe4** [iptv-org/community#851] [Q&A]
### Channel ID

NewEpsilonTV.gr

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by ztreamhd** [comment]
Neo TV (576p): https://edge1.mystream.gr/video427009/live/playlist.m3u8

**Comment by AntiPontifex** [comment]
Added for next update.

**Comment by freearhey** [comment]
https://edge1.mystream.gr/video427009/live/playlist.m3u8 (offline)
<img width="416" alt="image" src="https://user-images.githubusercontent.com/7253922/232238639-e29daa52-f260-47c5-b713-d71077ad964c.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19846
